docs/faq/proof-of-stake/general.md

```diff
@@ -1,98 +1,82 @@
+It is important to note that you are only assigning voting rights to the pool, not your actual funds. A VSP does not have access to your funds at any time.
 
 ---
 
 #### 4. What is the "ticket price"? 
 
-The price for tickets is determined by an algorithm that aims to keep the ticket pool size, which is the total amount of tickets in the PoS system ticket pool around a target size of 40960 tickets.
-The ticket price will go up or down according to the demand for tickets, and the number of tickets currently in the pool.
+The price for tickets is determined by an algorithm that aims to keep the ticket pool size, which is the total amount of tickets in the PoS system ticket pool, around a target size of 40960 tickets. This target was selected prior to Decred's launch as a way of balancing the average time until tickets are called to vote with the on-chain footprint of the voting process.
 
-Every 144 blocks the algorithm adjusts the ticket price. This is called a buying window. Each block can contain 20 newly bought tickets. This means that in every buying window a maximum of 2880 tickets
-can be added to the PoS system ticket pool.
+The ticket price goes up or down according to the demand for tickets, and the number of tickets currently in the pool. Every 144 blocks the algorithm adjusts the ticket price. This is called a buying window. Each block can contain 20 newly bought tickets. This means that in every buying window a maximum of 2880 tickets can be added to the PoS system ticket pool.
 
-The ticket price is always refunded no matter if your ticket votes, misses or expires.
+The ticket price is always refunded, no matter if your ticket votes, misses or expires.
 
 ---
 
 #### 5. What are fees? 
 
-The PoS system utilizes two types of fees, a txfee (also known as a 'split' fee) and a ticketfee.
+The PoS system utilizes two types of fees, a "txfee" (also known as a 'split' fee) and a "ticketfee".
 	
 The txfee is a fee you pay the network for handling the transaction to buy your ticket. This fee is at default set to a low amount (0.001 DCR/kB) and should not be changed.
 The ticketfee is the fee you pay if your ticket gets mined. This fee is an incentive for PoW miners to mine your ticket and add it to the ticket pool.
 
 In case the buying window is full, meaning there are more tickets being bought than there are available to be mined during that window, the ticketfee will be used to determine which tickets get put into
 the ticket pool. The tickets with the highest ticketfee will be selected by the PoW miners.
 
-Fees are calculated in DCR per kilobyte. Since a ticket is only about 300 (solo) or 540 (pool) bytes in size, the actual fees you pay will be lower than what you set as the specific fee amount.
+Fees are calculated in DCR per kilobyte. Since a ticket is only about 300 bytes (for solo) or 540 bytes (for pool) in size, the actual fees you pay will be lower than what you set as the specific fee amount.
```

Comment:
Replaced 'pool' with 'VSP':

> Since a ticket is only about 300 bytes (for solo) or 540 bytes (for VSP) in size, the actual fees you pay will be lower than what you set as the specific fee amount.